src/main/scala/io/moia/protos/teleproto/ReaderImpl.scala

```diff
@@ -417,7 +417,7 @@ object ReaderImpl {
     val protobufOptions = symbolsByTolerantName(c)(protobufType.typeSymbol.asClass.knownDirectSubclasses.filter(_.isModuleClass))
     val modelOptions    = symbolsByTolerantName(c)(modelType.typeSymbol.asClass.knownDirectSubclasses.filter(_.isModuleClass))
 
-    val unmatchedProtobufOptions = protobufOptions.filterKeys(name => !modelOptions.contains(name)).values.map(_.name.decodedName)
+    val unmatchedProtobufOptions = protobufOptions.filterKeys(name => !modelOptions.contains(name)).toMap.values.map(_.name.decodedName)
```

Comment:
```suggestion
    val unmatchedProtobufOptions =
      protobufOptions.toList.collect { case (name, v) if !modelOptions.contains(name) => v.name.decodedName }
```

Reply:
We could do it like this :point_up: and maybe avoid the deprecation warnings and also turn on the fatal warnings again

Reply:
Nice, done in c51dc1f